ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -28,7 +28,8 @@
           allow-whitelist-orgs-as-admins: true
           org-list:
             - ceph
-          trigger-phrase: ''
+          trigger-phrase: ''    
+          skip-build-phrase: 'qa tests .* skip build'
```

Comment:
I just googled and it showed me an example with regex(http://yaks-all-the-way-down.com/2016/04/22/Presenting-Git-Hub-Pull-Request-Builder.html), but we stick to single word, we should also add QA to that string just to be explicit that its QA changes QA-SKIP-TEST probably?